applications/asset_tracker_v2/src/modules/cloud_module.c

```diff
@@ -100,9 +100,18 @@ static int connect_retries;
 static struct cloud_data_cfg copy_cfg;
 const k_tid_t cloud_module_thread;
 
-/* Register message IDs that are used with the QoS library. */
-QOS_MESSAGE_TYPES_REGISTER(GENERIC, BATCH, UI, NEIGHBOR_CELLS, WIFI_ACCESS_POINTS,
-			   AGPS_REQUEST, PGPS_REQUEST, CONFIG, MEMFAULT);
+/* Message IDs that are used with the QoS library. */
+enum {
+	GENERIC = 0,
```

Comment:
Is it nessecary to explicitly set `= 0` here? Isn't the first element always 0.

Reply:
Just due to habit and the personal preference for being explicit.